model/infinispan/src/main/java/org/keycloak/models/sessions/infinispan/changes/PersistentSessionsChangelogBasedTransaction.java

```diff
@@ -66,46 +63,49 @@ public PersistentSessionsChangelogBasedTransaction(KeycloakSession session,
                                                        SerializeExecutionsByKey<K> serializerOffline) {
         kcSession = session;
 
-        if (!Profile.isFeatureEnabled(Profile.Feature.PERSISTENT_USER_SESSIONS)) {
```

Comment:
Why is this removed?

Reply:
While this is a safeguard, IMHO it is never happening as we're creating this transaction only when persistent user sessions are enabled.